src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1,47 +1,44 @@
-# Connect a data stack
-In this guide, you will learn how to connect an existing data stack to your application. This job includes understanding the required client configuration and how to connect the data stack. We will also review functions to create, read, update, and delete your data using the Amplify GraphQL client. Finally, we will look at how to subscribe to real-time events to watch for mutations in your data.
-[comment]: # (HEATHER, let's discuss how best to define "job" when we reference it the first time.)
+> *This guide provides essential information to help you select and complete the activities you need for your Amplify project. We organized each section around a distinct job or decision point to help you understand your available options, steps to complete, and recommended best practices. For users who want to learn more, we included extra details and explanations in expandable sections within the guide.*
 
-## Review the required configuration
 
-In this job, you will configure your client to point to the AWS AppSync API endpoint. You will first need to understand how to configure the Amplify API category and update the authentication type to complete this job. If you did not set up the data stack yourself, knowing the differences between authentication types can be helpful.
+In this guide, you will learn how to connect an existing data stack to your application. This job includes understanding the required client configuration and how to connect the data stack. We will also review functions to create, read, update, and delete your data using the Amplify GraphQL client. Finally, we will look at how to subscribe to real-time events to watch for mutations in your data.
 
-> **Before you begin, you will need:**
-> - An AWS AppSync API already set up. For guidance on how to build a GraphQL API, see [Designing a GraphQL API in AWS AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html)
-> - An application set up. For instructions on setting up an app, see [Project Setup: Create your application](https://docs.amplify.aws/lib/project-setup/create-application/q/platform/js/)
+Before you begin, you will need:
```

Comment:
Note - When reviewing the Auth guide we determined it made more sense to have prereqs at the top as it was getting confusing list and repeating them for each section. So I moved this up and other prereqs now reference the section that needs to be completed rather than repeating steps. Updated this guide with that change so we can keep the template consistent - if it does not work we are opening to revisiting!